mission_control/navigator_launch/launch/simulation.launch

```diff
@@ -38,6 +38,11 @@
         <param name="autorepeat_rate" value="5"/>
     </node>
 
+    <node pkg="navigator_emergency_control" type="navigator_emergency_control" name="navigator_emergency_xbee">
```

Comment:
Ah, I didn't fully understand which files the controller node should launched in. I tried to add it in the same locations as the node for the xbox controller, and that was simulation and master_shore. I figured the shore wouldn't launch with the boat itself, so I didn't add it there, but I guess I forgot to actually put it somewhere it'll launch with the boat. I think the most appropriate place would be hardware_drivers, so I'll add it there.

I've been using the simulator to test the output of wrench/cmd. Ever since I updated to dev a while ago, the simulator didn't run properly, but I wasn't sure if there was just something wrong with Gazebo locally. Are you saying the simulator is malfunctioning for everyone?